Runtime/Scripts/RGBotLocalRuntime/RG.cs

```diff
@@ -235,7 +235,136 @@ internal List<RGValidationResult> FlushValidations()
             return result;
         }
         
+        // Debug actions for bots
+
+        /**
+         * Draws a line from this agent's position to a position in the scene with the given color. The name can be
+         * used to update and remove this line later.
+         * <remarks>Gizmos must be enabled to see this line.</remarks>
+         * <param name="position">The position in the world to draw the line toward</param>
+         * <param name="color">The color of the line</param>
+         * <param name="name">A name to reference this line by for later operations</param>
+         * <example><code>rgObject.DrawLineTo(target.position, Color.red, "TargetEnemy");</code></example>
+         * <seealso cref="RemoveLineTo"/>
+         * <seealso cref="RemoveAllLinesTo"/>
+         */
+        public void DrawLineTo(Vector3 position, Color color, string name)
+        {
+            PerformAction(new RGActionRequest("DrawLineTo", new Dictionary<string, object>()
+            {
+                { "position", position},
+                { "color", color},
+                { "name", name},
+            }));
+        }
+
+        /**
+         * Removes a line that was drawn with DrawLineTo using its identifier
+         * <param name="name">The name of the line that was drawn</param>
+         * <seealso cref="DrawLineTo"/>
+         * <seealso cref="RemoveAllLinesTo"/>
+         */
+        public void RemoveLineTo(string name)
+        {
+            PerformAction(new RGActionRequest("DrawLineTo", new Dictionary<string, object>()
+            {
+                { "remove", true},
+                { "name", name},
+            }));
+        }
         
+        /**
+         * Removes all lines drawn using DrawLineTo
+         * <seealso cref="DrawLineTo"/>
+         * <seealso cref="RemoveLineTo"/>
+         */
+        public void RemoveAllLinesTo()
+        {
+            PerformAction(new RGActionRequest("DrawLineTo", new Dictionary<string, object>()
+            {
+                { "removeAll", true}
+            }));
+        }
+
+        /**
+         * Draws a snippet of text above this agent. Optionally change the offset which defines how far
```

Comment:
Yup my bad - I'm curious about going back to why we made this decision... in my mind they are two different things (i.e. an entity with actions and controlled by a client is different from just an entity sitting in the scene providing state). Thoughts? Maybe a discussion for another time, but I do think this could confuse users.